STYLEGUIDE.md

```diff
@@ -188,12 +188,22 @@ STATES = ["draft", "open", "closed"]
 STATES = %w(draft open closed)
 ```
 
+* Prefer `%i` to the literal array syntax when you need an array of symbols (and you don't need to maintain Ruby 1.9 compatibility). Apply this rule only to arrays with two or more elements.
```

Comment:
Suggestion: remove sentence stating "Apply this rule only to arrays with two or more elements." If we prefer a particular syntax for array literals why maintain a special case for single-element arrays?